types/global.d.ts

```diff
@@ -20,6 +20,8 @@ declare class SentryObject extends Sentry {
 
   // Calls either startSession or endSession based on optin status
   toggleSession: () => void;
+
+  captureException: (error: Error, extra?: Record<string, unknown>) => void;
```

Comment:
Do we need to add this? It should already be part of the Sentry type being extended here.

Reply:
If we remove it, we get errors like this:
```
app/scripts/migrations/082.ts:32:20 - error TS2339: Property 'captureException' does not exist on type 'SentryObject'.

32     global.sentry?.captureException?.(
                      ~~~~~~~~~~~~~~~~
```

Reply:
It looks like this is because this whole object is declared incorrectly. It's currently declared as a class, but it's not a class. It's an object. It should be defined like

```
type SentryObject = Sentry & {
  ...
}
```

Then we wouldn't need to re-add each method from Sentry itself that we're using.

Reply:
fixed in 494db5cb2f